### PR TITLE
Diffusers version fix

### PR DESCRIPTION
**Pull request by bubbliiiing**
1. Update diffusers version to 0.23.0，borrow from kohya_ss implementation
2. backward compatible with 0.18.2
3. tested

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def is_installed(package: str):
     print("Installing requirements for easyphoto-webui")
     launch.run_pip("install modelscope==1.9.3", "requirements for modelscope")
 
-if not is_installed("diffusers==0.18.2"):
+if not is_installed("diffusers"):
     print("Installing requirements for easyphoto-webui")
     launch.run_pip("install diffusers==0.18.2", "requirements for diffusers")
```
Comment (hkunzhe) [inline install.py:75]: 这里对应要修改成 install diffusers

File: install.py
Comment (hkunzhe) [inline install.py:1]: 还要补充 einops 这个依赖

**Comment by hkunzhe** [comment]
SDXL training tested by hkz.

**Comment by hkunzhe** [comment]
Backward compatibility tested by hkz.

**Review by wuziheng — APPROVED** [review]
LGTM

**Comment by wuziheng** [comment]
https://loc.dingtalk.com/L1VzZXJzL3d6aC9MaWJyYXJ5L0FwcGxpY2F0aW9uIFN1cHBvcnQvaURpbmdUYWxrLzE3MjQxNzE5NF92Mi9JbWFnZUZpbGVzLzIxMDAxLzE3MDE4NDI1MzY3MDBf5oiq5bGPMjAyMy0xMi0wNiDkuIvljYgyLjAyLjEzLnBuZw==?renderWidth=1746&renderHeight=722&renderOrientation=1&isLocal=1

run_pip(<=0.23.0) 是不能正常运行的

**Comment by wuziheng** [comment]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/4afaaf8a020c1df457bcf7250cb1c7f609699fa7/modules/launch_utils.py#L133

seems fault with  <=

**Comment by wuziheng** [comment]
test pass with train 、inference， without video

**Pull request merged**